samples/text_panes/components/text_pane.js

```diff
@@ -0,0 +1,97 @@
+AFRAME.registerComponent("text_pane", {
+  schema: {
+    jsonData: {
+      parse: JSON.parse,
+      stringify: JSON.stringify,
+    },
+  },
+
+  multiple: true,
+
+  init: function () {
+    const data = this.data.jsonData;
+    const el = this.el;
+
+    // Create textLabel
+    this.textLabel = document.createElement("a-text");
+    this.textLabel.setAttribute("id", "text");
+    this.textLabel.setAttribute("value", data.text[data.currPosition]);
+    this.textLabel.setAttribute("negate", "true");
+    this.textLabel.setAttribute("scale", "2 2 1");
+    this.textLabel.setAttribute("position", "-4.9 0 0.25");
+    this.el.appendChild(this.textLabel);
+
+    if (data.text.length > 1) {
+      const leftNavProp =
+        '{"width": "1.5", "height": "1.5", "depth": "0.01", "color": "white", "x": "-8", "y": "0", "z": "0.005"}';
+      const rightNavProp =
+        '{"width": "1.5", "height": "1.5", "depth": "0.01", "color": "white", "x": "8", "y": "0", "z": "0.005"}';
+
+      // Create left nav textLabel
+      this.buttonLabelLeft = document.createElement("a-text");
+      this.buttonLabelLeft.setAttribute("id", "text-nav-left");
+      this.buttonLabelLeft.setAttribute("value", "Prev");
+      this.buttonLabelLeft.setAttribute("negate", "true");
+      this.buttonLabelLeft.setAttribute("scale", "2 2 1");
+      this.buttonLabelLeft.setAttribute("color", "black");
+      this.buttonLabelLeft.setAttribute("position", "-8.5 0 0.01");
+      this.el.appendChild(this.buttonLabelLeft);
+
+      // Create left nav button
+      this.leftNav = document.createElement("a-entity");
+      this.leftNav.setAttribute("id", "nav-button-left");
+      this.leftNav.setAttribute("prism", "jsonData", leftNavProp);
+      this.leftNav.addEventListener("click", function () {
```

Comment:
Ran into a small bug, where clicking on the text directly doesn't cause the click to be registered (like clicking on `Next`, but clicking on the white part does. I believe this is because the text is slightly in front of the button, and only the button has a click handler. This bug should be solved by my previous comment though.

Reply:
Thanks for catching that! The bug should be resolved now